src/librustc_resolve/lib.rs

```diff
@@ -130,6 +130,17 @@ pub struct ParentScope<'a> {
     derives: Vec<ast::Path>,
 }
 
+impl<'a> ParentScope<'a> {
+    pub fn default(module: Module<'a>) -> ParentScope<'a> {
```

Comment:
Defeault with parameter looks odd. Perhaps `module_scope`?

Reply:
I agree that default isn't great here, but I'm not sure what to call it. Either way, I feel like a comment for this function would be helpful.

Reply:
Renamed into `ParentScope::module` and some comments are added.